modules/bar/workspaces/index.ts

```diff
@@ -199,9 +206,10 @@ const Workspaces = (monitor = -1, ws = 8) => {
                                             options.bar.workspaces.icons.available.bind("value"),
                                             options.bar.workspaces.icons.active.bind("value"),
                                             options.bar.workspaces.icons.occupied.bind("value"),
+                                            workspaceMask.bind(),
```

Comment:
```suggestion
                                            workspaceMask.bind("value"),
```
Let's bind the value of the variable here.